src/MSONOneOfParser.cc

```diff
@@ -10,21 +10,22 @@
 #include "MSONTypeSectionParser.h"
 #include "MSONPropertyMemberParser.h"
 
-namespace snowcrash {
+namespace snowcrash
+{
 
     /** Implementation of processNestedSection */
     MarkdownNodeIterator SectionProcessor<mson::OneOf>::processNestedSection(const MarkdownNodeIterator& node,
-                                                                             const MarkdownNodes& siblings,
-                                                                             SectionParserData& pd,
-                                                                             const ParseResultRef<mson::OneOf>& out) {
+        const MarkdownNodes& siblings,
+        SectionParserData& pd,
+        const ParseResultRef<mson::OneOf>& out)
+    {
 
         MarkdownNodeIterator cur = node;
         mson::Element element;
         SourceMap<mson::Element> elementSM;
 
         switch (pd.sectionContext()) {
-            case MSONMixinSectionType:
-            {
+            case MSONMixinSectionType: {
```

Comment:
It's possible, but with current configuration options we could achieve this only by breaking after all control statements. That's probably not what we want, though.